core/schema/service.go

```diff
@@ -193,6 +193,14 @@ func (s *Service) identifySchemaChange(ctx context.Context, request *changedetec
 		return fmt.Errorf("got error while identifying schema change for namespace : %s, schema: %s, version: %d, %s", request.NamespaceID, request.SchemaName, request.Version, err)
 	}
 	log.Printf("schema change result %s", sce.String())
+	return s.SendNotification(ctx, sce, request)
+}
+
+func (s *Service) SendNotification(ctx context.Context, sce *stencilv1beta1.SchemaChangedEvent, request *changedetector.ChangeRequest) error {
+	schemaID, err := s.repo.GetSchemaID(ctx, request.NamespaceID, request.SchemaName)
+	if err != nil {
+		return errors.New(fmt.Sprintf("got error while getting schema ID from DB %s", err.Error()))
```

Comment:
use `fmt.Errorf` which serves the purpose. Formats the string and creates error object.